sdk/src/trace/tracer_provider.cc

```diff
@@ -23,6 +23,9 @@ nostd::shared_ptr<opentelemetry::trace::Tracer> TracerProvider::GetTracer(
     nostd::string_view library_name,
     nostd::string_view library_version) noexcept
 {
+  if (library_name.data() == nullptr) {
+      library_name = "";
+  }
   // if (library_name == "") {
```

Comment:
Do you still intend keeping the previous `TODO`, since now the code intentionally forces that condition?

Reply:
The TODO is still needed I think, as the spec requires message reporting. Yes, I assume it is internal SDK logging.

>message reporting that the specified value is invalid SHOULD be logged